src/util/lp/factorization.h

```diff
@@ -0,0 +1,125 @@
+
+  /*++
+  Copyright (c) 2017 Microsoft Corporation
+
+  Module Name:
+
+  <name>
+
+  Abstract:
+
+  <abstract>
+
+  Author:
+  Nikolaj Bjorner (nbjorner)
```

Comment:
did I actually contribute to this file?

Reply:
I believe there were some discussions about it with you.